<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1397</width>
    <height>686</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Beertabs</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QDateEdit" name="drinkDateInput">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>241</width>
      <height>22</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::WheelFocus</enum>
    </property>
    <property name="displayFormat">
     <string>MMM/dd/yyyy</string>
    </property>
    <property name="calendarPopup">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="drinkDateLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>221</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Date</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QComboBox" name="breweryInput">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>66</y>
      <width>361</width>
      <height>30</height>
     </rect>
    </property>
    <property name="editable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="breweryLabel">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>50</y>
      <width>341</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Brewery</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="typeLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>221</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Type</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QComboBox" name="typeInput">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>126</y>
      <width>241</width>
      <height>30</height>
     </rect>
    </property>
    <property name="editable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QComboBox" name="nameInput">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>126</y>
      <width>361</width>
      <height>30</height>
     </rect>
    </property>
    <property name="editable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="nameLabel">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>110</y>
      <width>341</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Name</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="abvInput">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>190</y>
      <width>62</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="ibuInput">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>190</y>
      <width>62</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="abvLabel">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>170</y>
      <width>58</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>ABV</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="ibuLabel">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>170</y>
      <width>58</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>IBU</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="sizeLabel">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>170</y>
      <width>58</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Size</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QTextEdit" name="notesInput">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>190</y>
      <width>231</width>
      <height>131</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="notesLabel">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>170</y>
      <width>231</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Notes</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QTableWidget" name="drinkLogTable">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>350</y>
      <width>1371</width>
      <height>291</height>
     </rect>
    </property>
    <row>
     <property name="text">
      <string>1</string>
     </property>
    </row>
    <column>
     <property name="text">
      <string>Date</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Name</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Type</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Brewery</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>ABV</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>IBU</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Size</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Rating</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>ID</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Timestamp</string>
     </property>
    </column>
    <item row="0" column="0">
     <property name="text">
      <string>09/07/2020</string>
     </property>
    </item>
    <item row="0" column="1">
     <property name="text">
      <string>Everything Rhymes with Orange</string>
     </property>
    </item>
    <item row="0" column="2">
     <property name="text">
      <string>IPA</string>
     </property>
    </item>
    <item row="0" column="3">
     <property name="text">
      <string>Roughtail Brewing</string>
     </property>
    </item>
    <item row="0" column="4">
     <property name="text">
      <string>8.0</string>
     </property>
    </item>
    <item row="0" column="5">
     <property name="text">
      <string>65</string>
     </property>
    </item>
    <item row="0" column="6">
     <property name="text">
      <string>12</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="drinkLogLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>330</y>
      <width>1371</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Drink Log</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="drinksThisWeekLabel">
    <property name="geometry">
     <rect>
      <x>773</x>
      <y>60</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Drinks this week:</string>
    </property>
   </widget>
   <widget class="QLabel" name="ozAlcoholConsumedLabel">
    <property name="geometry">
     <rect>
      <x>773</x>
      <y>90</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Oz. alcohol consumed:</string>
    </property>
   </widget>
   <widget class="QLabel" name="ozAlcoholRemainingLabel">
    <property name="geometry">
     <rect>
      <x>773</x>
      <y>120</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Oz. alcohol remaining:</string>
    </property>
   </widget>
   <widget class="QLabel" name="favoriteBreweryLabel">
    <property name="geometry">
     <rect>
      <x>773</x>
      <y>150</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Favorite brewery:</string>
    </property>
   </widget>
   <widget class="QLabel" name="favoriteBeerLabel">
    <property name="geometry">
     <rect>
      <x>773</x>
      <y>180</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Favorite beer:</string>
    </property>
   </widget>
   <widget class="QLabel" name="drinksLeftLabel">
    <property name="geometry">
     <rect>
      <x>983</x>
      <y>60</y>
      <width>171</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Std. drinks left for the week:</string>
    </property>
   </widget>
   <widget class="QLabel" name="avgAbvDrinkLabel">
    <property name="geometry">
     <rect>
      <x>983</x>
      <y>90</y>
      <width>161</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Average ABV per drink:</string>
    </property>
   </widget>
   <widget class="QLabel" name="averageIbuDrinkLabel">
    <property name="geometry">
     <rect>
      <x>983</x>
      <y>120</y>
      <width>161</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Average IBU per drink:</string>
    </property>
   </widget>
   <widget class="QLabel" name="drinksThisWeekOutput">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>60</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QLabel" name="ozAlcoholConsumedOutput">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>90</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QLabel" name="ozAlcoholRemainingOutput">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>120</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QLabel" name="favoriteBreweryOutput">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>150</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QLabel" name="favoriteBeerOutput">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>180</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QLabel" name="drinksLeftOutput">
    <property name="geometry">
     <rect>
      <x>1170</x>
      <y>60</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QLabel" name="avgAbvDrinkOutput">
    <property name="geometry">
     <rect>
      <x>1170</x>
      <y>90</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QLabel" name="avgIbuDrinkOutput">
    <property name="geometry">
     <rect>
      <x>1170</x>
      <y>120</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>10</y>
      <width>20</width>
      <height>311</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLabel" name="statsLabel">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>20</y>
      <width>551</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Stats</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="drinkDataLabel">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>20</y>
      <width>551</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Drink Data</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="submitRowButton">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>290</y>
      <width>112</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Submit</string>
    </property>
   </widget>
   <widget class="QPushButton" name="clearFieldsButton">
    <property name="geometry">
     <rect>
      <x>415</x>
      <y>290</y>
      <width>112</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear</string>
    </property>
   </widget>
   <widget class="QPushButton" name="deleteRowButton">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>290</y>
      <width>112</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete Row</string>
    </property>
   </widget>
   <widget class="QComboBox" name="filterCategoryInput">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>290</y>
      <width>251</width>
      <height>32</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="filterTextInput">
    <property name="geometry">
     <rect>
      <x>1040</x>
      <y>290</y>
      <width>341</width>
      <height>32</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="filterCategoryLabel">
    <property name="geometry">
     <rect>
      <x>767</x>
      <y>275</y>
      <width>241</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Filter Category</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="filterLabel">
    <property name="geometry">
     <rect>
      <x>1050</x>
      <y>275</y>
      <width>321</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Filter</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QSpinBox" name="ratingInput">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>240</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>10</number>
    </property>
   </widget>
   <widget class="QLabel" name="ratingLabel">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>220</y>
      <width>58</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Rating</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QSpinBox" name="sizeInput">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>190</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1397</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>drinkDateInput</tabstop>
  <tabstop>breweryInput</tabstop>
  <tabstop>typeInput</tabstop>
  <tabstop>nameInput</tabstop>
  <tabstop>abvInput</tabstop>
  <tabstop>ibuInput</tabstop>
  <tabstop>sizeInput</tabstop>
  <tabstop>ratingInput</tabstop>
  <tabstop>deleteRowButton</tabstop>
  <tabstop>clearFieldsButton</tabstop>
  <tabstop>submitRowButton</tabstop>
  <tabstop>filterCategoryInput</tabstop>
  <tabstop>filterTextInput</tabstop>
  <tabstop>notesInput</tabstop>
  <tabstop>drinkLogTable</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
